app/src/main/kotlin/com/intive/picover/common/di/AppModule.kt

```diff
@@ -24,7 +24,7 @@ object AppModule {
 
 	@Provides
 	fun providePartiesRepository() =
-		PartiesRepository(Firebase.firestore)
+		PartiesRepository(dev.gitlive.firebase.Firebase.firestore)
```

Comment:
full package needed due to mix with regular firebase package. To be simplified when `Firebase.storage.reference` will be migrated as well